src/status_im/events.cljs

```diff
@@ -1101,10 +1101,34 @@
  :pairing.ui/pair-devices-pressed
  []
  (fn [cofx _]
-   (pairing/start cofx)))
+   (pairing/pair-installation cofx)))
 
 (handlers/register-handler-fx
  :pairing.ui/synchronize-installation-pressed
  []
  (fn [cofx _]
    (pairing/send-installation-message cofx)))
+
+(handlers/register-handler-fx
+ :pairing.ui/enable-installation-pressed
+ []
+ (fn [cofx [_ installation-id]]
```

Comment:
is `[]` as argument is really needed here?

Reply:
not needed, thanks :)